test-page/src/views/Category.vue

```diff
@@ -117,8 +117,12 @@
           <SfProductCard
             v-for="(product, i) in products"
             :key="i"
-            :title="product.name"
-            image="/img/product_thumb.png"
+            :title="product.name || ''"
```

Comment:
Not a fix request but quick insight.

Getting null names (or any kind of basic data, that is nullable) from Shopware might be surprising for the new storefront developers. Maybe there is some way to extract that logic from the template to a method? When we were working on interfaces for Shopware models we saw so many nullable fields. I have a feeling we will need to add `|| ''` in many places, so if that would be possible we should cover these null appearances in some eye-catching layer to make it easy to find later.

Reply:
It's more like fix we need to add into StorefrontUI, it showed an error, when got null value here :) i just puted it here to avoid errors on dev console